grpc.go

```diff
@@ -108,20 +114,23 @@ func (s *server) by(method string, fn func() (string, error)) (*cacher.Hardware,
 
 // ByMAC implements cacher.CacherServer
 func (s *server) ByMAC(ctx context.Context, in *cacher.GetRequest) (*cacher.Hardware, error) {
+	trace.SpanFromContext(ctx).SetAttributes(attribute.String("MAC", in.MAC))
```

Comment:
Non of the By* queries have something like `trace.SpanFromContext(ctx).AddEvent("By*")` was that on purpose?

Reply:
Yeah, I wasn't sure whether to do that or not. It's probably a good idea. We can add in follow-ups.